tests/test_fix_cf.py

```
@@ -0,0 +1,95 @@
+            # --- WARM UP PHASE ---
+            print("\n🔥 Warming up browser session...")
+            
+            # 1. Visit Google first
+            print("   Visiting Google...")
+            page.goto("https://www.google.co.uk", wait_until="domcontentloaded")
+            time.sleep(random.uniform(3.0, 5.0))
+            
+            # 2. Simulate human activity
+            try:
+                print("   Simulating human activity...")
+                page.mouse.move(random.randint(100, 500), random.randint(100, 500), steps=20)
```

Ruff (0.14.7)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)